Documentation/ApiOverview/LinkBrowser/Linkhandler/Index.rst

```diff
@@ -230,3 +221,13 @@ detail page you can do it like this:
          additionalParams.wrap = &tx_news_pi1[controller]=News&tx_news_pi1[action]=detail&tx_news_pi1[news]=|
       }
    }
+
+
+..  toctree::
+    :titlesonly:
+    :hidden:
```

Comment:
Why hidden?

Reply:
Because it should be displayed in the menu on the left but not inside the page in this position. We do also have sufficient manual links to those pages, so one more list of them doesn't make sense here